x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -35,106 +37,121 @@ export const EnginesList: React.FC = () => {
-          defaultMessage: 'Engines',
-        }),
-        rightSideItems: [
-          <EuiButton
-            fill
-            iconType="plusInCircle"
-            data-test-subj="enterpriseSearchContentEnginesCreationButton"
-            href={'TODO'}
-          >
-            {i18n.translate('xpack.enterpriseSearch.content.engines.createEngineButtonLabel', {
-              defaultMessage: 'Create engine',
-            })}
-          </EuiButton>,
-        ],
-      }}
-      pageViewTelemetry="Engines"
-      isLoading={false}
-    >
-      <EuiText>
-        {i18n.translate('xpack.enterpriseSearch.content.engines.description', {
-          defaultMessage:
-            'Engines allow you to query indexed data with a complete set of relevance, analytics and personalization tools. To learn more about how engines work in Enterprise search ',
-        })}
-
-        <EuiLink data-test-subj="documentationLink" href="TODO" target="_blank">
-          {i18n.translate('xpack.enterpriseSearch.content.engines.documentation', {
-            defaultMessage: 'explore our Engines documentation',
-          })}
-        </EuiLink>
-      </EuiText>
-      <EuiSpacer />
-      <div>
-        <EuiFieldSearch
-          value={searchQuery}
-          placeholder={i18n.translate('xpack.enterpriseSearch.content.engines.searchPlaceholder', {
-            defaultMessage: 'Search engines',
-          })}
-          aria-label={i18n.translate('xpack.enterpriseSearch.content.engines.searchBar.ariaLabel', {
-            defaultMessage: 'Search engines',
+    <>
+      <DeleteEngineModal />
+      <EnterpriseSearchContentPageTemplate
+        pageChrome={[
+          i18n.translate('xpack.enterpriseSearch.content.engines.breadcrumb', {
+            defaultMessage: 'Engines',
+          }),
+        ]}
+        pageHeader={{
+          pageTitle: i18n.translate('xpack.enterpriseSearch.content.engines.title', {
+            defaultMessage: 'Engines',
+          }),
+          rightSideItems: [
+            <EuiButton
+              fill
+              iconType="plusInCircle"
+              data-test-subj="enterpriseSearchContentEnginesCreationButton"
+              href={'TODO'}
+            >
+              {i18n.translate('xpack.enterpriseSearch.content.engines.createEngineButtonLabel', {
+                defaultMessage: 'Create engine',
+              })}
+            </EuiButton>,
+          ],
+        }}
+        pageViewTelemetry="Engines"
+        isLoading={false}
+      >
+        <EuiText>
+          <FormattedMessage
+            id="xpack.enterpriseSearch.content.engines.description"
+            defaultMessage="Engines allow you to query indexed data with a complete set of relevance, analytics and personalization tools. To learn more about how engines work in Enterprise search {documentationUrl}"
+            values={{
+              documentationUrl: (
+                <EuiLink data-test-subj="documentationLink" href="TODO" target="_blank">
```

Comment:
I _think_ `data-test-subj` should be globally unique. so `enginesDocumentationLink` or `engines-documentation-link` if kabab-case is preferred.

Reply:
I think `enginesDocumentationLink` as it aligns with other values in file

Reply:
Data-test-subj doesn't really need to be globally unique, though it may help.